src/components/flex-resizer/flex-resizer.tsx

```diff
@@ -213,6 +213,8 @@ export class FlexResizer {
 
   @Watch("disabled")
   watchDisabled(disabled: boolean) {
+    this.a.style["flex-grow"] = null;
+    this.b.style["flex-grow"] = null;
```

Comment:
I doubt this should be done always it should be cleared only if `disabled == true`... when it is `false` the previous value should be returned....